.vscode/launch.sample.json

```diff
@@ -0,0 +1,22 @@
+{
+  "configurations": [
+    {
+      "type": "node",
+      "name": "vscode-jest-tests.v2",
+      "request": "launch",
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "cwd": "${workspaceFolder}",
+      "runtimeExecutable": "yarn",
+      "args": [
+        "test",
```

Comment:
We already defined `test` in package.json and we want to use it to run the test, hence we set `yarn` as `runtimeExecutable` and `test` as the first argument.